### PR TITLE
Apply Staff/Part property changes when moving between parts with arrow buttons

### PR DESCRIPTION
**Pull request by EightSmart**
Resolves: #25494

Previously, when moving between parts in the Edit Staff/Part property dialog using the arrow buttons, your changes would immediately be discarded with no warning. As suggested in #25494, the arrow buttons now apply your changes before moving to the next part. The relevant tooltips have also been updated.

- [x] I signed the [CLA](https://musescore.org/en/cla)
- [x] The title of the PR describes the problem it addresses
- [x] Each commit's message describes its purpose and effects, and references the issue it resolves
- [ ] If changes are extensive, there is a sequence of easily reviewable commits
- [x] The code in the PR follows [the coding rules](https://github.com/musescore/MuseScore/wiki/CodeGuidelines)
- [x] There are no unnecessary changes
- [x] The code compiles and runs on my machine, preferably after each commit individually
- [ ] I created a unit test or vtest to verify the changes I made (if applicable)

### CODE REVIEW

File: src/notation/view/widgets/editstaff.ui [hunk 1]
```diff
@@ -1031,10 +1031,10 @@
         </size>
        </property>
        <property name="toolTip">
-        <string>Go to previous staff</string>
+        <string>Apply and go to previous staff</string>
        </property>
        <property name="accessibleName">
-        <string>Go to previous staff</string>
+        <string>Apply and go to previous staff</string>
```
Comment (Jojo-Schmitz) [inline src/notation/view/widgets/editstaff.ui:1037]: Question: is an accessibleName needed when there is a toolTip (with the same content? Wouldn't screen readers read it twice? @shoogle ?

**Review by mathesoncalum — APPROVED** [review]
Looks good to me - thanks!

I don't think [this](https://github.com/musescore/MuseScore/pull/25605/files#r1853687158) is a blocker as it was also the case before this PR. If @shoogle feels this is worth addressing we can do it separately.

**Comment by Jojo-Schmitz** [comment]
No, it certainly isn't a blocker

**Comment by cbjeukendrup** [comment]
@zacjansheski That might be the ideal behaviour indeed. However, the justification for the current behaviour is (that it is easier to implement and) that when you click the "Apply" button and then "Cancel", then nothing is undone because you already hit "Apply"; and the tooltip for these arrow buttons is supposed to indicate that clicking those buttons implies clicking "Apply".  

Anyway, it could be addressed later indeed, if anyone gets to that before the entire dialog is overhauled at some point.

**Pull request merged**